src/QMCHamiltonians/ACForce.cpp

```diff
@@ -15,18 +15,23 @@
  */
 #include "ACForce.h"
 #include <sstream>
+#include "OhmmsData/AttributeSet.h"
 
 namespace qmcplusplus
 {
 ACForce::ACForce(ParticleSet& source, ParticleSet& target, TrialWaveFunction& psi_in, QMCHamiltonian& H)
-    : ions(source), elns(target), psi(psi_in), ham(H), FirstForceIndex(-1), Nions(0)
+    : ions(source), elns(target), psi(psi_in), ham(H), FirstForceIndex(-1), Nions(0), useSpaceWarp(false), swt(target,source)
```

Comment:
Does ACForce modify ions? If no, it is better to set `ions` and `source` as const reference.

Reply:
ACForce potentially modifies ions, if QMCHamiltonian.evaluateIonDerivs starts doing finite differences or something.  Would be complicated to make this const here.  For SpaceWarpTransformation, there's no good reason the ions should be modified, so I'll make those consts.

Reply:
That makes sense.